Tools/cases_generator/generate_cases.py

```diff
@@ -641,17 +641,14 @@ def write_abstract_interpreter_instructions(
             for thing in self.everything:
                 match thing:
                     case OverriddenInstructionPlaceHolder():
-                        # TODO: Is this helpful?
-                        self.write_overridden_instr_place_holder(thing)
+                        pass
                     case parsing.InstDef():
                         instr = AbstractInstruction(self.instrs[thing.name].inst)
                         if instr.is_viable_uop() and instr.name not in SPECIALLY_HANDLED_ABSTRACT_INSTR:
                             self.out.emit("")
                             with self.out.block(f"case {thing.name}:"):
                                 instr.write(self.out, tier=TIER_TWO)
```

Comment:
I wonder if you even need the `AbstractInstruction` class. Maybe you could just call a different method on `Instruction`?

Reply:
I was thinking of how we might need to expand on it more in the future, so a separate class might be better.

Reply:
When that happens in the future you can refactor the code. Until then, I recommend less code.

As you may have noticed this code gets refactored a lot. :-) It's easy because the generator is not a public API -- all we care about is whether it generates a handful of files correctly from bytecodes.c at build time. But I worry about copying and pasting code, because that's harder to refactor.